package/SUSEConnect.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Sun Jul 21 10:00:00 UTC 2021 - Natnael Getahun <ngetahun@suse.com>
+
+- Update to 0.3.31
+- Disallow registering via SUSEConnect if SUMA is present.
```

Comment:
Suggestion: 
`- Disallow registering via SUSEConnect if the system is managed by SUSE Manager`